class/.htaccess

```diff
@@ -0,0 +1,3 @@
+Options +FollowSymLinks
+RewriteEngine on
+RewriteRule ^class/esc/.* https://raw.githubusercontent.com/hbz/vocabs-edu/master/esc.ttl [R=302,L]
```

Comment:
This probably isn't what you want.  It will redirect URLs like https://w3id.org/class/class/esc/.